dev/validate_bom__test.py

```diff
@@ -861,9 +946,35 @@ def make_test_command_or_none(self, test_name, spec, metric_labels):
         'python', test_path,
         '--log_dir', citest_log_dir,
         '--log_filebase', test_name,
-        '--native_host', 'localhost',
-        '--native_port', str(self.__forwarded_ports[microservice_api].port)
     ]
+    if microservice_api in self.__outbound_service_configs:
+      service_config = self.__outbound_service_configs[microservice_api]
+      host = service_config['host']
+      command.extend([
+          '--native_host', host
+      ])
+      if host.startswith('http://'):
+        command.extend([
+            '--native_port', '80'
+        ])
+      elif host.startswith('https://'):
+        command.extend([
+            '--native_port', '443'
+        ])
+      if 'bearer_auth_token' in service_config:
+        command.extend([
```

Comment:
It is not ideal to be passing this on the command line from a security perspective, though would be ok especially since the token has a relatively short lifetime.

I dont think citest supports options from a file. If not then I'll add it later. But since you are adding this parameter in now, how about support passing it through an environment variable?